src/libpgagroal/prometheus.c

```diff
@@ -2402,6 +2960,13 @@ metrics_cache_finalize(void)
    }
 
    now = time(NULL);
-   cache->valid_until = now + config->metrics_cache_max_age;
+   cache->valid_until = now + config->common.metrics_cache_max_age;
    return cache->valid_until > now;
 }
+
+static bool
+is_prometheus_enabled(void)
+{
+   struct configuration* config = (struct configuration*)shmem;
```

Comment:
add test against nulls?

Reply:
Not required as we initiate `prometheus` memory allocation only if `config->metrics` > 0.

Reply:
> Not required as we initiate `prometheus` memory allocation only if `config->metrics` > 0.

Well, but if we export the function to somewhere else, we could need some extra (redundant) checks.

Reply:
Okay, will add the `NULL` check for `prometheus` to avoid any unlikely cases that may occur.